### PR TITLE
[CALCITE-5408] Return type of PERCENTILE_CONT should be the same as sort expression

### PR DESCRIPTION
**Pull request by zoudan**
Return type of PERCENTILE_CONT should be the same as sort expression

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java [hunk 1]
```diff
@@ -2195,7 +2195,7 @@ public class SqlStdOperatorTable extends ReflectiveSqlOperatorTable {
    */
   public static final SqlAggFunction PERCENTILE_CONT =
       SqlBasicAggFunction
-          .create(SqlKind.PERCENTILE_CONT, ReturnTypes.DOUBLE,
```
Comment (libenchao) [inline core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java:2198]: the comment should also be changed.

**Review by libenchao — COMMENTED** [review]
@zoudan Thanks for your PR, I only left one minor comment.

File: core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java (hunk 1 above)
Comment (zoudan) [reply core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java:2198]: @libenchao Thanks for reviewing, I have resolved it just now.

**Review by libenchao — APPROVED** [review]
LGTM, will merge after the CI passes. Thanks for your contribution @zoudan

**Pull request closed without merging**